staging/cse/windows/nvidiagpudriverfunc.ps1

```diff
@@ -0,0 +1,337 @@
+function Start-InstallGPUDriver {
+  $result = @{
```

Comment:
You can use $RebootNeeded = $false, set it to $true later and return $RebootNeeded in the last line.